resources/k8s/docker_build.sh

```diff
@@ -0,0 +1,326 @@
+#!/bin/bash
+
+# This script is responsible for creating a Docker Image of the given Product with WUM updates and
+# Pushing it to the Private Docker registry
+
+set -o xtrace
+
+IFS='' read -r -d '' wum_repo_info<<"EOF"
+defaultchannel: full
+repositories:
+  wso2:
+    enabled: false
+    name: WSO2 Update Repository
+    url: https://api.updates.wso2.com
+    tokenurl: https://api.updates.wso2.com/token
+    appkey: WUNMR2hnaGdIU25FajB4SngzNkRSeDFOT1pVYTp5enB4SWd6bWpncjVlWWNkdXFhblpYc2JCRXNh
+    refreshtoken: cf5daccb-8939-3b2c-a187-34441a12743d
+    accesstoken: 5b7bd3df-5df2-33f8-bb68-4f5e39433141
+  uat:
+    enabled: true
+    name: WSO2 UAT Repository
+    url: https://gateway.api.cloud.wso2.com/t/wso2umuat
+    tokenurl: https://api.updates.wso2.com/token
+    appkey: R0dnZThYMmk2T2E2ZldjbHhKWWplTV93REJFYTo5Q0FGbG1oR09ZbjRhTzkyNFp5REh6VEZFeTBh
+    refreshtoken: cf5daccb-8939-3b2c-a187-34441a12743d
+    accesstoken: 5b7bd3df-5df2-33f8-bb68-4f5e39433141
+EOF
+
+SERVICE_ACCOUNT="gke-bot@testgrid.iam.gserviceaccount.com"
+CLUSTER_NAME="chathurangi-test-cluster"
+ZONE="us-central1-a"
+PROJECT_NAME="testgrid"
+REG_LOCATION="asia.gcr.io"
+PROJECT_ID="testgrid/wso2-docker"
```

Comment:
These need to be environment variables because we may need to customize them. CLUSTER_NAME is also dependent on whether we run this in testgrid dev vs prod deployment.

Parameterization is a core part of this effort.

Reply:
Done